docs/en/operations/settings/settings-formats.md

```diff
@@ -634,6 +634,23 @@ Ignore extra columns in rows with more columns than expected and treat missing c
 
 Disabled by default.
 
+### output_format_markdown_escape_special_characters {#output_format_markdown_escape_special_characters}
+
+When enabled, escape special characters in Markdown.
+
+[CommonMark](https://spec.commonmark.org/0.30/#example-12) defines the following special characters that can be escaped:
+
+```
+\! \" \# \$ \% \& \' \( \) \* \+ \, \- \. \/ \: \; \< \= \> \? \@ \[ \\ \] \^ \_ \` \{ \| \} \~
```

Comment:
Minor: No need for escaping in a markdown quote block, please change to `! " # [...]`.

Also, please mention that `\` is used for escaping - asking because in Markdown afaik allows HTML which has & escaping.